Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -364,10 +367,8 @@ class BrowserViewController: UIViewController {
         }
 
         webViewContainerToolbar.snp_makeConstraints { make in
-            make.trailing.equalTo(webViewContainer)
-            make.leading.equalTo(webViewContainer)
+            make.leading.trailing.top.equalTo(webViewContainer)
```

Comment:
Just have a habit of using `leading`/`trailing` over `left`/`right` by default because of RTL compatibility, but `left`/`right` is fine with me.